service-base/src/main/java/org/eclipse/hono/service/AbstractProtocolAdapterBase.java

```diff
@@ -168,8 +168,7 @@ private void onDisconnectMessaging(final ProtonConnection con) {
             messaging.connect(createClientOptions(), connectAttempt -> {
                 if (connectAttempt.succeeded()) {
                     LOG.debug("reconnected to Hono Messaging");
-                }
-                else {
+                } else {
                     LOG.warn("cannot reconnect to Hono Messaging");
```

Comment:
IMHO this should be logged at debug level as well. This is a totally expected situation and we are already handling it by means of trying to re-connect. In production, there will be multiple instances of which at least one will probably be connected at all times. What we might want to do is introduce a counter for unsuccessful re-connect attempts and e.g. issue a WARN statement after, say, 100 unsuccessful attempts. But we should do that in another PR.
WDYT?